libs/react-hooks/src/use-tiles-state/use-tiles-state.ts

```diff
@@ -20,7 +20,7 @@ export function useTilesState(map?: Map) {
   }, [setTilesLoading])
 
   const onLoad = useCallback(
-    (e: MapSourceDataEvent) => {
+    (e: any) => {
```

Comment:
I don't find the `useTilesState` hooks being used anywhere. I'd remove it if it's not used so we don't have to maintain it.

Reply:
I'd leave it because we might want to back to the idea of showing a loading per tile in the miniglobe... it was unstable with the previous fork but I think we can improve it now